static/app/views/replays/deadRageClick/deadRageSelectorCards.tsx

```diff
@@ -17,10 +18,17 @@ function DeadRageSelectorCards() {
   const location = useLocation();
 
   return (
-    <SplitCardContainer>
-      <DeadClickTable location={location} />
-      <RageClickTable location={location} />
-    </SplitCardContainer>
+    <ShowHidePanel
+      headerShow={() => t('Show Actionable Replay Insights')}
+      headerHide={() => t('Hide Actionable Replay Insights')}
+      content={() => (
+        <SplitCardContainer>
```

Comment:
These are recreated on each render so it means this component will always rerender :)